### PR TITLE
Please add my contribution

### PR DESCRIPTION
**Pull request by jivanbhardwaj**
this is 2nd contribution to open source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ Join us [here](https://discord.com/invite/nNtVfKddDD).
 - [Shawn Charles](https://twitter.com/ShawnBasquiat)
 - [Benny Yen](https://twitter.com/benny123tw/)
 - [Rupali Haldiya](https://twitter.com/rupali_codes)
+- [Jivan Bhardwaj](https://github.com/jivanbhardwaj)
```
Comment (ykdojo) [inline README.md:57]: can you put your name at the end?

**Review by ykdojo — CHANGES_REQUESTED** [review]

**Comment by jivanbhardwaj** [comment]
Okay .. I'll do that .

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ Join us [here](https://discord.com/invite/nNtVfKddDD).
 - [Dunsin](https://twitter.com/DunsinWebDev)
 - [banf](https://twitter.com/banf)
 - [JAX](https://twitter.com/44jax44)
-- [Muhammad Adnan Yousaf](https://twitter.com/ma593y)
+i- [Muhammad Adnan Yousaf](https://twitter.com/ma593y)
```
Comment (ykdojo) [inline README.md:47]: please fix this line

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jivanbhardwaj) [reply README.md]: Done!

**Review by subhoghoshX — APPROVED** [review]

**Comment by jivanbhardwaj** [comment]
Thanks, @subhoghoshX @ykdojo finally get good about open source contribution.

**Comment by subhoghoshX** [comment]
> Thanks, @subhoghoshX @ykdojo finally get good about open source contribution.

Best wishes on your Opensource adventure.

**Pull request merged**